src/dsp_tools/commands/excel2json/models/input_error.py

```diff
@@ -240,3 +241,44 @@ def execute_error_protocol(self) -> str:
         if self.message_path:
             msg.append(f"The error occurred at {self.message_path}")
         return separator.join(msg)
+
+
+@dataclass(frozen=True)
+class ListNodeProblem:
+    node_id: str
+    problems: dict[str, str]
+
+    def execute_error_protocol(self) -> str:
+        msg = [
+            f"The node '{self.node_id}' has the following problem(s):",
+        ]
+        msg.extend([f"Field: '{key}' Problem: {value}" for key, value in self.problems.items()])
```

Comment:
and similarly in the other occurrences

```suggestion
        msg.extend([f"Field: '{key}', Problem: {value}" for key, value in self.problems.items()])
```